tests/e2e/specs/tracks/view-edit-track.test.ts

```diff
@@ -0,0 +1,66 @@
+import {expect} from '@wdio/globals';
+import {describe, it, before} from 'mocha';
+import {byTextMatches, byResourceId, byText} from '../../utils/selectors';
+
+describe('Tracks - View and Edit Saved Track', () => {
+  before(async () => {
+    const obsListTab = await $('~Go to ObservationsList');
```

Comment:
Requires updating the label where it's used in the UI.

```suggestion
    const obsListTab = await $('~Go to observations list.');
```

Guess whether "observations" should be a proper noun in our case is debatable. I would say it shouldn't be but not overly opinionated on it.

Reply:
Realizing that updating this is maybe a bit out of scope for this PR - otherwise there would be a couple of other introduced tests in this PR that would benefit from similar suggestions. Okay with these being done as a follow-up.